datasets/met-office/collection/met-office-uk-deterministic-near-surface-level/template.json

```diff
@@ -0,0 +1,420 @@
+{
+    "id": "met-office-uk-deterministic-near-surface-level",
+    "type": "Collection",
+    "links": [
+        {
+            "rel": "license",
+            "href": "https://creativecommons.org/licenses/by-sa/4.0/deed.en",
+            "type": "text/html",
+            "title": "Creative Commons Attribution-ShareAlike 4.0"
+        },
+        {
+            "rel": "describedBy",
+            "href": "https://www.metoffice.gov.uk/services/data/external-data-channels",
+            "title": "Met Office Dataset Documentation"
+        }
+    ],
+    "title": "Near-surface level collection Met Office UKV 2km deterministic forecast",
+    "assets": {
+        "thumbnail": {
+            "href": "",
+            "type": "image/png",
+            "roles": [
+                "thumbnail"
+            ],
+            "title": "Near-surface level collection Met Office UKV 2km deterministic forecast Thumbnail"
+        }
+    },
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    15,
+                    50,
+                    5,
+                    62
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2024-01-01T00:00:00Z",
+                    null
+                ]
+            ]
+        }
+    },
+    "license": "proprietary",
+    "keywords": [
+        "Met Office",
+        "Weather",
+        "Forecast",
+        "UK",
+        "Precipitation",
+        "Temperature",
+        "Wind",
+        "Pressure",
+        "Humidity"
+    ],
+    "providers": [
+        {
+            "url": "https://www.metoffice.gov.uk/",
+            "name": "Met Office",
+            "roles": [
+                "producer",
+                "licensor",
+                "processor"
+            ]
+        },
+        {
+            "url": "https://planetarycomputer.microsoft.com",
+            "name": "Microsoft",
+            "roles": [
+                "host",
+                "processor"
+            ]
+        }
+    ],
+    "item_assets": {
+        "fog_fraction_at_screen_level": {
+            "type": "application/netcdf",
+            "title": "Fog fraction at screen level",
+            "description": "Here fog means a visibility of 1000 m or lower. The reduction in visibility is caused by  water droplets or minute ice crystals forming close to the surface. This quantity represents the fraction of horizontal  grid square occupied by fog. An alternative interpretation is that this represents the fractional probability of fog being present at any location in the grid square.",
+            "unit": "1",
+            "height": "1.5m",
```

Comment:
Not quite sure about this one...if it's a dataset-specific value, we may want to use a custom "extension" attribute, e.g. `met_office_deterministic:height`. There's also other values that reference "at screen level" but don't have a height, so maybe this should just be excluded?